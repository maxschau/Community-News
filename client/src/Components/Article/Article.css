body {
    margin: 0;
    padding: 0;
}

#container {
    width: 60%;
    margin: auto;
}

#container h1 {
    text-align: center;

}

.timeText {
    font-size: 80%;
}

#img {
    width: 100%;
}


@media only screen and (max-width: 600px) {
    #container {
        width: 100%;
    }
}