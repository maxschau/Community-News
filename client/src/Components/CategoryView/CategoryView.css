.main {
    margin: auto;
    text-align: center;
    border: 1px solid black;
    height: auto;
    width: 60%;
}

@media only screen and (max-width: 600px) {
    .main {
        width: 100%;
    }
}


