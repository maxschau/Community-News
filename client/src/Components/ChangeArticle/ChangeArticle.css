#containerReg {
    width: 50%;
    margin: auto;
}

@media only screen and (max-width: 600px) {
    #containerReg {
        width: 100%;
    }
}