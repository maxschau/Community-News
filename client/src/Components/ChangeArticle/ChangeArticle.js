// @flow

import React, {Component} from 'react'
import {Row, Column} from '../widgets';
import ArticleService, {Article} from '../../services/ArticleService';
import CategoryService from '../../services/CategoryService';
import {toast} from 'react-toastify';
import { confirmAlert } from 'react-confirm-alert'; // Import
import 'react-confirm-alert/src/react-confirm-alert.css'; // Import css

//Setting the types of the state
type State = {
    headline : string,
    ingress : string,
    contents : string,
    category: number,
    image : string,
    importance: number,
    time : string, 
    author : string,
    likes : number
}

type Props = {
    id : number
}

class ChangeArticle extends Component<State, Props> {
    constructor(props : any) {
        super(props);

        this.state = {
            headline:"",
            ingress : "",
            contents:"",
            category:"",
            image:"",
            importance:"",
            time:"",
            author : "",
            categories: []
        }
    }

    submitDelete = () => {
        confirmAlert({
        title: 'Bekreftelse av sletting',
        message: 'Er du sikker på at du vil slette artikkelen?',
        buttons: [
            {
            label: 'Ja',
            onClick : () => this.delete()
            },
            {
            label: 'Nei'
            }
        ]
        });
    }

    submitSave = () => {
        confirmAlert({
        title: 'Bekreftelse av oppdatering',
        message: 'Er du sikker på at du vil oppdatere artikkelen?',
        buttons: [
            {
            label: 'Ja',
            onClick : () => this.save()
            },
            {
            label: 'Nei'
            }
        ]
        });
    }

    notifyChange: void = () => {
        toast("Endring av artikkel vellykket", {type: toast.TYPE.SUCCESS, position: toast.POSITION.BOTTOM_LEFT});
    }

    notifyDelete: void = () => toast("Sletting gikk bra", {type: toast.TYPE.SUCCESS, position: toast.POSITION.BOTTOM_LEFT});

    save() : void {
        let articleService = new ArticleService();
        let newArticle = new Article(this.state.headline, this.state.ingress, this.state.contents, this.state.category, this.state.image, this.state.importance, this.state.author);
        articleService.updateOneArticle(this.props.match.params.id, newArticle)
            .then(() => {
                this.notifyChange();
                window.location.hash = '/articles/' + this.props.match.params.id
            })
            .catch((error) => console.error(error))
    }

    delete() : void {
        let articleService = new ArticleService();
        articleService.deleteOneArticle(this.props.match.params.id)
            .then(() => {
                this.notifyDelete();
                window.location.hash = '/home';
            })
            .catch((error) => console.error(error))
    }

    handleChange : void = (e) => {
        this.setState({
            [e.target.name]: e.target.value
        });
    };

    render() {
        return(
            <div id = "containerReg">
                <form>
                    <div className="form-group">
                        <label>Overskrift: </label>
                        <input type="text" className="form-control"
                               id="fname" placeholder="Skriv inn overskrift"
                               value = {this.state.headline}
                               name="headline" onChange={this.handleChange} required/>
                    </div>
                    <div className="form-group">
                        <label>Ingress:</label>
                        <textarea  className="form-control" id="lname" value = {this.state.ingress} rows="2"
                                   placeholder="Skriv inn innholdet" name="ingress" onChange={this.handleChange} required/>
                    </div>

                    <div className="form-group">
                        <label>Innhold:</label>
                        <textarea  className="form-control" id="lname" value = {this.state.contents} rows="3"
                               placeholder="Skriv inn innholdet" name="contents" onChange={this.handleChange} required/>
                    </div>


                    <div className="form-group">
                        <label>Bilde:</label>
                        <input type="text" className="form-control"
                               id="fname" value = {this.state.image} placeholder="Skriv inn bilde-URL"
                               name="image" onChange={this.handleChange} required/>
                    </div>
                    <div className="form-group">
                        <Row>
                            <Column>
                                <label>Kategori: </label>
                            </Column>
                        </Row>
                        <Row>
                            <Column>
                                <select className="btn btn-secondary" name="category" value= {this.state.category} onChange={this.handleChange}>
                                {this.state.categories.map((categories) => {
                                    return(
                                        <option value={categories.id} name = "category" selected={this.state.category === categories.id}>{categories.name}</option>
                                    )
                                })}
                                </select>
                            </Column>
                        </Row>
                        <Row>
                            <Column>
                                <label>Viktighet </label>
                            </Column>
                        </Row>
                        <Row>
                            <Column>
                                <label>1<input type="radio" name="importance" value={1} checked= {this.state.importance === 1} onChange={this.handleChange}/></label>
                                <label>2<input type="radio" name="importance" value={2} checked= {this.state.importance === 2} onChange={this.handleChange}/></label>

                            </Column>
                        </Row>
                    </div>
                    <Row>
                        <Column>
                            <button type="button" className="btn btn-secondary" onClick={() => this.submitSave()}>
                            Lagre
                            </button>
                        </Column>
                        <Column>
                            <button type="button" className="btn btn-danger" onClick={() => this.submitDelete() }> Slett </button>
                        </Column>
                    </Row>
                </form>
            </div>
        );
    }



    componentDidMount() {
            let articleService = new ArticleService();
            let categoryService = new CategoryService();

            categoryService.getAll()
                .then((categories) => {
                    this.setState({
                        categories: categories.data
                    })
                })
                .catch((error) => {
                    console.error(error);
                });

            articleService.getOneArticle(this.props.match.params.id)
                .then((article) => {
                    this.setState({
                        headline: article.data[0].headline,
                        ingress: article.data[0].ingress,
                        contents: article.data[0].contents,
                        category: article.data[0].category,
                        image: article.data[0].image,
                        importance: article.data[0].importance,
                        time: article.data[0].time,
                        author: article.data[0].author
                    });
                    //window.scrollTo(0, document.body.scrollHeight)
                })
        
    }
}

export default ChangeArticle;