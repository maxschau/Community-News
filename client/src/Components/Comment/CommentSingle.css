.commentCard {
    background-color: #F4F4F4;
}

.commentCard p {
    padding-left: 2px;
}

.text {
    padding-left: 4px;
}

.minorText {
    font-size: 12px;
    text-align: right;
}