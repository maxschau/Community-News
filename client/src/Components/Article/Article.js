// @flow
import React, {Component} from 'react';
import ArticleService from '../../services/ArticleService';
import {Column, Row, Card} from '../../Components/widgets';
import {Link} from "react-router-dom";
import {faThumbsUp, faThumbsDown }  from "@fortawesome/free-solid-svg-icons"
import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
import './Article.css';
import {toast} from 'react-toastify';


type State = {
    headline: string,
    ingress: string,
    contents: string,
    category: number,
    image: string,
    importance:number,
    time: string,
    author: string,
    likes : number,
    liked : boolean
}

type Props = {}

type props= {}

class Article extends Component<Props> {
    constructor(props : any) {
        super(props);
        this.state = {
            headline:"",
            ingress: "",
            contents:"",
            category:"",
            image:"",
            importance:"",
            time:"",
            author:"",
            likes : "",
            liked : false
        };
    }

    notifyFailure: void = () => toast("Det er kun lov å gi én like!", {type: toast.TYPE.ERROR, position: toast.POSITION.BOTTOM_LEFT});

    handleThumbsUp = () => {
        if (!(this.state.liked)) {
            this.setState({
                likes : this.state.likes + 1,
                liked : true
            });
        this.updateLikes();
        } else {
            this.notifyFailure();
        }
    };

    updateLikes = () => {
        let articleService = new ArticleService();
        articleService.updateLikes(this.props.match.params.id)
            .then(() => {
                console.log("Updating likes")
            })
            .catch((error) => {
                console.error(error);
            })
    };

    render() {
        return(
            <div className="d-flex justify-content-center" id="container">
                <Card title={""}>
                    <Row>
                        <Column> 
                            <img src={this.state.image} alt = {this.state.headline} id="img" />
                        </Column>
                    </Row>
                    <Row>
                        <Column>
                            <h1>{this.state.headline}</h1>
                        </Column>
                    </Row>
                    <Row>
                        <Column>
                            <h4>{this.state.ingress}</h4>
                            <hr />
                        </Column>
                    </Row>
                    <Row>
                        <Column width={9}>
                            <h5 className={"timeText float-left"}><b>Publisert: </b> {this.state.time} av {this.state.author}</h5>
                        </Column>
                        <Column width={3}>
                            <span><p onClick={() => this.handleThumbsUp()}><FontAwesomeIcon id="iconLikes" icon={faThumbsUp}/>   {this.state.likes}</p></span>
                        </Column>
                    </Row>
                    <Row>
                        <Column>
                            <h5>{this.state.contents}</h5>
                        </Column>
                    </Row>

                    <Row>
                        <Column>
                            <Link to={"/articles/edit/" + this.props.match.params.id}><button className="btn btn-secondary">Endre </button></Link>
                        </Column>
                    </Row>
                </Card>
            </div>
        );
    }


    componentDidMount() {
        let articleService = new ArticleService();
        articleService.getOneArticle(this.props.match.params.id)
            .then((article) => {
                console.log(article.data[0]);
                let time = article.data[0].time.substring(0, article.data[0].time.length-1);
                time = time.replace("T", " ");
                time = time.replace('.000', "");
                this.setState({
                    headline : article.data[0].headline,
                    ingress: article.data[0].ingress,
                    contents:article.data[0].contents,
                    category:article.data[0].category,
                    image:article.data[0].image,
                    importance:article.data[0].importance,
                    time: time,
                    author: article.data[0].author,
                    likes : article.data[0].likes
                });
            })
            .catch((error => console.error(error)))

    }
}

export default Article;