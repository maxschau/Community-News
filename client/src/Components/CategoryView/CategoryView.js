// @flow

import React, {Component} from 'react';
import {Row, Column} from '../widgets'
import {ArticleService, Article} from '../../services/ArticleService';
import OtherArticle from "../FrontPage/OtherArticle";
import './CategoryView.css';
import CategoryService , {Category} from '../../services/CategoryService';
import {faForward, faBackward}  from "@fortawesome/free-solid-svg-icons";
import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";


type State = {
    id : number,
    articles : Article[],
    category : Category,
    pageNumber : number,
    maxPage : number,
    limitPerPage : number
}

type Props = {
    match : { params : { id: number}}
}
class CategoryView extends Component<Props, State> {

    constructor(props : any) {
        super(props);
        this.state = {
            id : -1,
            articles: [],
            category: new Category(""),
            pageNumber: 0,
            maxPage : -1,
            limitPerPage : 6
        }
    }
    render() {
        return (
            <div className="main">
                <Row>
                    <Column>
                        {<h1>{this.state.category.name}</h1>}
                    </Column>
                </Row>
                <hr />
                <Row>
                    {this.state.articles.slice(this.state.pageNumber*this.state.limitPerPage, this.state.pageNumber*this.state.limitPerPage+6).map((article) => {
                        return <OtherArticle key={article.id} id={article.id} headline={article.headline} image={article.image}/>
                    })}
                </Row>
                <hr />
                <Row>
                    <Column>
                        <p onClick={this.handleLastPage}><FontAwesomeIcon icon={faBackward} /> </p>
                    </Column>
                    <Column>
                        <p>Page {this.state.pageNumber + 1}</p>
                    </Column>
                    <Column>
                        <p onClick={this.handleNextPage}><FontAwesomeIcon icon={faForward} /> </p>
                    </Column>
                </Row>
            </div>
        );
    }

    handleNextPage  = () => {
        let number = (this.state.pageNumber + 1)%(this.state.maxPage);
        this.setState({
            pageNumber : number
        })
    };

    handleLastPage = () => {
        if (this.state.pageNumber > 0) {
            let number = this.state.pageNumber - 1;
            this.setState({
                pageNumber : number
            })
        }
    };


    componentDidUpdate(prevProps : Props, prevState : State) : void {
        if (this.state.pageNumber !== prevState.pageNumber) {
            console.log("WTF IS GOING ON, THE STATE HAS CHANGED")
        }
        if (this.props.match.params.id !== prevProps.match.params.id) {
            let articleService = new ArticleService();
            let categoryService = new CategoryService();
            articleService.getArticlesByCategory(this.props.match.params.id)
                .then((articles) => {
                    this.setState({
                        id: this.props.match.params.id,
                        articles: articles,
                        maxPage : Math.ceil((articles.length / this.state.limitPerPage)),
                        pageNumber : 0
                    });
                })
                .catch((error) => console.error(error));
            categoryService.getOne(this.props.match.params.id)
                .then((category) => {
                    this.setState({
                    category: category[0]
                });
            });
        }
    }

    componentDidMount() {
        let articleService = new ArticleService();
        let categoryService = new CategoryService();
        articleService.getArticlesByCategory(this.props.match.params.id)
            .then((articles) => {
                this.setState({
                    id: this.props.match.params.id,
                    articles: articles,
                    maxPage : Math.ceil((articles.length / this.state.limitPerPage))
                });
            })
            .catch((error) => console.error(error));
        categoryService.getOne(this.props.match.params.id)
            .then((category) => {
                console.log(category);
                this.setState({
                    category: category[0]
                });
            })
            .catch((error) => console.error(error));
    }
}

export default CategoryView;