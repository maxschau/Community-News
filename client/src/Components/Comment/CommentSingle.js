// @flow

import React, {Component} from 'react';
import {Column, Row} from '../../Components/widgets';
import './CommentSingle.css';

type Props = {
    name : string,
    comment : string
}

class CommentSingle extends Component<Props> {
    render() {
        return(
            <div>
                <Row>
                    <Column>
                        <b>{this.props.name}:</b>
                    </Column>
                </Row>
                <Row>
                    <Column>
                        {this.props.comment}
                    </Column>
                </Row>
                 <hr />
            </div>
        )
    }

}

export default CommentSingle;