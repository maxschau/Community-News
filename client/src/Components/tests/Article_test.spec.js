import React from 'react';
import ReactDOM from 'react-dom';


import renderer from 'react-test-renderer';

import Article from '../Article/Article';
import CategoryView from '../CategoryView/CategoryView';
import ChangeArticle from '../ChangeArticle/ChangeArticle';
import CommentSingle from '../Comment/CommentSingle';
import CommentView from '../Comment/CommentView';
import CreateArticle from '../CreateArticle/CreateArticle';
import Footer from '../Footer/Footer';
import FrontPage from '../FrontPage/FrontPage';
import MainArticle from '../FrontPage/MainArticle';
import OtherArticle from '../FrontPage/OtherArticle';
import NavBar from '../Header/NavBar/NavBar';
import Header from '../Header/Header';
import LiveFeedElement from '../LiveFeed/LiveFeedElement';
import LiveFeed from '../LiveFeed/LiveFeed';


describe('Article renders correctly', () => {
    test('snapshot renders', () => {
        const props = {
            match: { params: {id: 1}}
        }
        const params = {id : 1};
        const component = renderer.create(<Article {...props}/>);
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    })
})

describe('Footer renders correctly', () => {
    test('snapshot renders', () => {
        const component = renderer.create(<Footer />);
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});

/*

test('Article component renders without crashing', () => {
    const div = document.createElement('div');
    const props = {
        match: {params : {id: 1}}
    };
    const params = {id : 1};
    ReactDOM.render(<Article {...props} />, div);
});


test("CategoryView component renders without crashing", () => {
    const div = document.createElement('div');
    const props = {
        match: {params : {id: 1}}
    };
    const params = {id : 1};
    ReactDOM.render(<CategoryView {...props} />, div);
});

test("ChangeArticle component renders without crashing", () => {
    const div = document.createElement('div');
    const props = {
        match: {params : {id: 1}}
    };
    const params = {id : 1};
    ReactDOM.render(<ChangeArticle {...props} />, div);
});

test("CommentSingle component renders without crashing", () => {
    const div = document.createElement('div');
    ReactDOM.render(<CommentSingle name="Test" comment="test test test"/>, div);
});

test("CommentView component renders without crashing", () => {
    const div = document.createElement('div');
    ReactDOM.render(<CommentView id={1} />, div);
});

test("CreateArticle component renders without crashing", () => {
    const div = document.createElement('div');
    ReactDOM.render(<CreateArticle />, div);
});

test("Footer component renders without crashing", () => {
    const div = document.createElement('div');
    ReactDOM.render(<Footer/>, div);
});

test("FrontPage component renders without crashing", () => {
    const div = document.createElement('div');
    ReactDOM.render(<FrontPage/>, div);
});
*/







