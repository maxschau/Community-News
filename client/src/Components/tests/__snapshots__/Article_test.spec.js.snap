// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Article renders correctly snapshot renders 1`] = `
<div
  className="d-flex justify-content-center"
  id="container"
>
  <div
    className="card"
  >
    <div
      className="card-body"
    >
      <h5
        className="card-title"
      >
        
      </h5>
      <div
        className="card-text"
      >
        <div
          className="row"
        >
          <div
            className="col"
          >
            <img
              alt=""
              id="img"
              src=""
            />
          </div>
        </div>
        <div
          className="row"
        >
          <div
            className="col"
          >
            <h1>
              
            </h1>
          </div>
        </div>
        <div
          className="row"
        >
          <div
            className="col"
          >
            <h4>
              
            </h4>
            <hr />
          </div>
        </div>
        <div
          className="row"
        >
          <div
            className="col-9"
          >
            <h5
              className="timeText float-left"
            >
              <b>
                Publisert: 
              </b>
               
              
               av 
              
            </h5>
          </div>
          <div
            className="col-3"
          >
            <span>
              <p
                data-placement="top"
                data-toggle="tooltip"
                onClick={[Function]}
                title="Gi en like!!"
              >
                <svg
                  aria-hidden="true"
                  className="svg-inline--fa fa-thumbs-up fa-w-16 "
                  data-icon="thumbs-up"
                  data-prefix="fas"
                  focusable="false"
                  id="iconLikes"
                  role="img"
                  style={Object {}}
                  viewBox="0 0 512 512"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M104 224H24c-13.255 0-24 10.745-24 24v240c0 13.255 10.745 24 24 24h80c13.255 0 24-10.745 24-24V248c0-13.255-10.745-24-24-24zM64 472c-13.255 0-24-10.745-24-24s10.745-24 24-24 24 10.745 24 24-10.745 24-24 24zM384 81.452c0 42.416-25.97 66.208-33.277 94.548h101.723c33.397 0 59.397 27.746 59.553 58.098.084 17.938-7.546 37.249-19.439 49.197l-.11.11c9.836 23.337 8.237 56.037-9.308 79.469 8.681 25.895-.069 57.704-16.382 74.757 4.298 17.598 2.244 32.575-6.148 44.632C440.202 511.587 389.616 512 346.839 512l-2.845-.001c-48.287-.017-87.806-17.598-119.56-31.725-15.957-7.099-36.821-15.887-52.651-16.178-6.54-.12-11.783-5.457-11.783-11.998v-213.77c0-3.2 1.282-6.271 3.558-8.521 39.614-39.144 56.648-80.587 89.117-113.111 14.804-14.832 20.188-37.236 25.393-58.902C282.515 39.293 291.817 0 312 0c24 0 72 8 72 81.452z"
                    fill="currentColor"
                    style={Object {}}
                  />
                </svg>
                   
                
              </p>
            </span>
          </div>
        </div>
        <div
          className="row"
        >
          <div
            className="col"
          >
            <h5>
              
            </h5>
          </div>
        </div>
        <div
          className="row"
        >
          <div
            className="col-true"
          >
            <a
              href="#/articles/edit/1"
              onClick={[Function]}
            >
              <button
                className="btn btn-dark"
              >
                Endre 
              </button>
            </a>
          </div>
        </div>
        <hr />
        <div
          className="row"
        >
          <div
            className="col"
          >
            <div>
              <div
                className="row"
              >
                <div
                  className="col"
                >
                  <h4>
                    Kommentarer
                  </h4>
                </div>
              </div>
              <div
                className="row"
              >
                <div
                  className="col"
                >
                  <button
                    className="btn btn-dark"
                    id="btnComment"
                    onClick={[Function]}
                  >
                    Skriv ny kommentar 
                  </button>
                  <hr />
                </div>
              </div>
              <div
                className="row"
              >
                <div
                  className="col"
                >
                  <div
                    id="containerNewComment"
                  >
                    <form>
                      <div
                        className="form-group"
                      >
                        <label>
                          Ditt navn:
                        </label>
                        <input
                          className="form-control"
                          id="inpName"
                          name="name"
                          placeholder="Skriv navnet ditt"
                          required={true}
                          type="text"
                        />
                      </div>
                      <div
                        className="form-group"
                      >
                        <label>
                          Din kommentar:
                        </label>
                        <textarea
                          className="form-control"
                          id="inpComment"
                          name="kommentar"
                          placeholder="Din kommentar..."
                          required={true}
                        />
                      </div>
                      <button
                        className="btn btn-dark"
                        onClick={[Function]}
                        type="button"
                      >
                        Lagre
                      </button>
                    </form>
                  </div>
                </div>
              </div>
              <div
                className="containerComments"
              />
              <div
                className="row"
              >
                <div
                  className="col"
                >
                  <p
                    onClick={[Function]}
                  >
                    Forrige
                  </p>
                </div>
                <div
                  className="col"
                >
                  1
                </div>
                <div
                  className="col"
                >
                  <p
                    onClick={[Function]}
                  >
                    Neste
                  </p>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Footer renders correctly snapshot renders 1`] = `
<div
  className="mainFooter row justify-content-center"
>
  <div
    className="mainFooter row justify-content-center"
  >
    <div
      id="col"
    >
      <a
        href="http://www.facebook.com"
        rel="noopener noreferrer"
        target="_blank"
      >
        <img
          alt="fbook"
          id="imgLogo"
          src="facebook.png"
        />
      </a>
      <a
        href="http://www.twitter.com"
        rel="noopener noreferrer"
        target="_blank"
      >
        <img
          alt="twitter"
          id="imgLogo"
          src="twitter.png"
        />
      </a>
      <a
        href="http://www.flickr.com"
        rel="noopener noreferrer"
        target="_blank"
      >
        <img
          alt="flickr"
          id="imgLogo"
          src="flickr.png"
        />
      </a>
      <a
        href="http://www.instagram.com"
        rel="noopener noreferrer"
        target="_blank"
      >
        <img
          alt="insta"
          id="imgLogo"
          src="instagram.png"
        />
      </a>
      <a
        href="http://www.youtube.com"
        rel="noopener noreferrer"
        target="_blank"
      >
        <img
          alt="youtube"
          id="imgLogo"
          src="youtube.png"
        />
      </a>
    </div>
  </div>
  <div
    className="row justify-content-center"
  >
    <div
      id="col"
    >
      <h5>
        Kontakt: 
      </h5>
      <span>
        <b>
          Tlf: 
        </b>
        91782159 
      </span>
      <br />
      <span>
        <b>
          E-post: 
        </b>
        redaksjonen@kalvskinnetNews.com 
      </span>
    </div>
  </div>
</div>
`;
